core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -302,6 +302,7 @@ market.trades.tooltip.candle.close=Close:
 market.trades.tooltip.candle.high=High:
 market.trades.tooltip.candle.low=Low:
 market.trades.tooltip.candle.average=Average:
+market.trades.tooltip.candle.average=Median:
```

Comment:
You used the same identifier as for average

Reply:
That is now corrected. Apologies.